resources/js/components/Room/HistoryComponent.vue

```diff
@@ -0,0 +1,420 @@
+      <b-modal :static="modalStatic" size="xl" hide-footer id="statsModal">
+        <template #modal-title>
+          <h5 v-if="statsMeeting">{{ $t('meetings.stats.modalTitle',{room: room.name }) }}
+          <br><small>{{ $date.utc(statsMeeting.start).tz(userTimezone).format('DD.MM.YY HH:mm') }} <raw-text>-</raw-text> {{ statsMeeting.end == null ? $t('meetings.now') : $date.utc(statsMeeting.end).tz(userTimezone).format('DD.MM.YY HH:mm') }}</small>
+          </h5>
+        </template>
+        <b-alert show variant="info"><i class="fas fa-info-circle"></i> {{ $t('meetings.stats.noBreakoutSupport')}}</b-alert>
+
+        <line-chart v-if="statsMeeting" :height="250" :chart-data="chartData" :chart-options="chartOptions"></line-chart>
+      </b-modal>
+      <!-- Attendance modal -->
+      <b-modal :static="modalStatic" size="xl" hide-footer id="attendanceModal" title-tag="div" title-class="w-100">
+        <template #modal-title >
+          <div class="d-flex justify-content-between align-items-center">
+            <h5 v-if="attendanceMeeting">{{ $t('meetings.attendance.modalTitle',{room: room.name}) }}
+              <br><small>{{ $date.utc(attendanceMeeting.start).tz(userTimezone).format('DD.MM.YY HH:mm') }} <raw-text>-</raw-text> {{ $date.utc(attendanceMeeting.end).tz(userTimezone).format('DD.MM.YY HH:mm') }}</small>
+            </h5>
+            <div v-if="attendanceMeeting"><b-button target="_blank" :href="'/download/attendance/'+attendanceMeeting.id" ><i class="fas fa-file-excel"></i> {{ $t('meetings.attendance.download') }}</b-button></div>
+          </div>
+        </template>
+        <b-alert show variant="info"><i class="fas fa-info-circle"></i> {{ $t('meetings.attendance.noBreakoutSupport')}}</b-alert>
+
+        <b-table
+          id='attendance-table'
+          :current-page="attendanceCurrentPage"
+          :per-page="settings('pagination_page_size')"
+          :fields="attendanceTableFields"
+          sort-by="name"
+          v-if="attendance"
+          :items="attendance"
+          hover
+          stacked="md"
+          show-empty
+        >
+          <!-- Show message on empty attendance list -->
+          <template v-slot:empty>
+            <i>{{ $t('meetings.attendance.nodata') }}</i>
+          </template>
+
+          <template v-slot:cell(email)="data">
+            {{ data.item.email || "---" }}
+          </template>
+
+          <template v-slot:cell(duration)="data">
+            {{ $t('meetings.attendance.durationMinute',{duration: data.item.duration}) }}
+          </template>
+
+          <template v-slot:cell(sessions)="data">
+            <p v-for="session in data.item.sessions" :key="session.id" >{{ $date.utc(session.join).tz(userTimezone).format('DD.MM.YY HH:mm') }} <raw-text>-</raw-text> {{ $date.utc(session.leave).tz(userTimezone).format('DD.MM.YY HH:mm') }} <raw-text>(</raw-text>{{ $t('meetings.attendance.durationMinute',{duration: session.duration})}}<raw-text>)</raw-text></p>
```

Comment:
We should use the locale date format with [$d](https://kazupon.github.io/vue-i18n/guide/datetime.html)!